src/app/components/TableWithPagination/TableWithPagination.tsx

```diff
@@ -81,18 +106,28 @@ export const TableWithPagination: FunctionComponent<
             variant="pagination"
             alignment={{ default: "alignRight" }}
           >
-            {getPagination(rows.length, false)}
+            {getPagination(false)}
           </ToolbarItem>
         </ToolbarContent>
       </Toolbar>
-      <Table
-        actionResolver={actionResolver}
-        ariaLabel={tableLabel}
-        columns={columns}
-        cssClasses="overview__table"
-        rows={rows}
-      />
-      {getPagination(rows.length, true)}
+      {isLoading ? (
```

Comment:
@vpellegrino there might be a merge conflict with https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/38